hummingbot/connector/exchange/btc_markets/btc_markets_order_book_tracker.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+import asyncio
+import bisect
+import logging
+import hummingbot.connector.exchange.btc_markets.btc_markets_constants as constants
+import time
+
+from collections import defaultdict, deque
+from typing import Optional, Dict, List, Deque
+from hummingbot.core.data_type.order_book_message import OrderBookMessageType
+from hummingbot.logger import HummingbotLogger
+from hummingbot.core.data_type.order_book_tracker import OrderBookTracker
+from hummingbot.connector.exchange.btc_markets.btc_markets_order_book_message import BtcMarketsOrderBookMessage
+from hummingbot.connector.exchange.btc_markets.btc_markets_active_order_tracker import BtcMarketsActiveOrderTracker
+from hummingbot.connector.exchange.btc_markets.btc_markets_api_order_book_data_source \
+    import BtcMarketsAPIOrderBookDataSource
+from hummingbot.connector.exchange.btc_markets.btc_markets_order_book import BtcMarketsOrderBook
+
+
+class BtcMarketsOrderBookTracker(OrderBookTracker):
+    _logger: Optional[HummingbotLogger] = None
+
+    @classmethod
+    def logger(cls) -> HummingbotLogger:
+        if cls._logger is None:
+            cls._logger = logging.getLogger(__name__)
+        return cls._logger
+
+    def __init__(self, trading_pairs: Optional[List[str]] = None,):
+        super().__init__(BtcMarketsAPIOrderBookDataSource(trading_pairs), trading_pairs)
+
+        self._ev_loop: asyncio.BaseEventLoop = asyncio.get_event_loop()
+        self._order_book_snapshot_stream: asyncio.Queue = asyncio.Queue()
+        self._order_book_diff_stream: asyncio.Queue = asyncio.Queue()
+        self._order_book_trade_stream: asyncio.Queue = asyncio.Queue()
+        self._process_msg_deque_task: Optional[asyncio.Task] = None
+        self._past_diffs_windows: Dict[str, Deque] = {}
+        self._order_books: Dict[str, BtcMarketsOrderBook] = {}
+        self._saved_message_queues: Dict[str, Deque[BtcMarketsOrderBookMessage]] = \
+            defaultdict(lambda: deque(maxlen=1000))
+        self._active_order_trackers: Dict[str, BtcMarketsActiveOrderTracker] = defaultdict(BtcMarketsActiveOrderTracker)
+        self._order_book_stream_listener_task: Optional[asyncio.Task] = None
+        self._order_book_trade_listener_task: Optional[asyncio.Task] = None
+
+    @property
+    def exchange_name(self) -> str:
+        """
+        Name of the current exchange
+        """
+        return constants.EXCHANGE_NAME
+
+    async def _track_single_book(self, trading_pair: str):
```

Comment:
The connector should not required a special logic for the method `_track_single_book`. Instead of creating a special order book tracker for the connector, please use an instance of `OrderBookTracker`

Reply:
Based on crypto.com connector - https://github.com/hummingbot/hummingbot/blob/master/hummingbot/connector/exchange/crypto_com/crypto_com_order_book_tracker.py#L69